openapi-specs/schema.json

```diff
@@ -860,6 +860,12 @@
             "shardId": {
                 "description": "The shard id holding the replica to be copied",
                 "type": "string"
+            },
+            "transferType": {
+              "description": "The type of transfer to perform: 'copy' or 'move'. Defaults to 'copy' if not specified.",
+              "type": "string",
+              "enum": ["copy", "move"],
+              "default": "copy"
```

Comment:
Should the capitalisation here be made consistent with that defined in `ReplicationReplicateDetailsReplicaResponse` below?